Add 44 sprites (with nine of them being Glowkate's actually, but I put it all in one sprites.png and sprites.js)

Biggest-Brian opened the pull request:
Made two sprites, one for Octopus Methology technology from the Abandoned Ocean, another for Overtime aspect.

Biggest-Brian committed:
Add overtime sprite

Overtime: "The Minute Hand gains 0.2s per second while disabled. Bonus time counts against your world-time when used"

Biggest-Brian committed:
Add Octopus Methodology sprite

Description before unlocking: 
"The octopus claim they know ways to improve their routines and machines."

Description after unlocking:
"Octopus Methodology: Octopuses can specialise in investigation, and octopuses work twice as efficiently."

History log:
"We have no idea what thought processes guide these cephalopod allies of ours, but they know how to get results."

Biggest-Brian committed:
correct octopus methodology name

It was starting with an uppercase letter

Biggest-Brian committed:
Delete octopus methodology.png

It was in the wrong place

Biggest-Brian committed:
Add octopus methodology sprite

Technology from Abandoned World

Description before unlocking: 
"The octopus claim they know ways to improve their routines and machines."

Description after unlocking:
"Octopus Methodology: Octopuses can specialise in investigation, and octopuses work twice as efficiently."

History log:
"We have no idea what thought processes guide these cephalopod allies of ours, but they know how to get results."

Toby222 commented:
Nice, now I only need to remember how to properly add sprites to the game

Biggest-Brian committed:
Add octal efficiency sprite

Description before unlocking: 
"The octopuses wish to further enhance their productivity for collective gain."

Description after unlocking:
"Octal Efficiency: Octopuses, investigators, and their machines are twice as effective. Find unity in efficiency."

History log:
"The instructions constructed and disseminated by the octopuses are complex and only understood to other octopuses. Head hurts. Something about the number eight."

Toby222 commented:
Ideally rename files to camelCase (octopusMethodology instead of "octopus methodology")

Biggest-Brian committed:
Fixing file names

Changing octal efficiency to octalEfficiency

Biggest-Brian committed:
Fixing file names

See previous change, this is the same thing

Biggest-Brian committed:
Adding src's

Last three sprites adding source files .kra (Krita extension).
Hopefully this makes a folder

Biggest-Brian committed:
Fixing file names

Biggest-Brian committed:
Fixing file names

Biggest-Brian committed:
Add destiny gamble sprite

I LOVE GAMBLING!!!!

Biggest-Brian committed:
add destiny gamble .kra

This is the source for the destiny gamble sprite. .kra is for Krita

Biggest-Brian commented:
Made another two sprites, this time for Octal Efficiency technology from the Abandoned Ocean and Destiny Gamble aspect. Added .kra (Krita extension) source img/src/BB's kra src

Biggest-Brian committed:
Add reverse engineering sprite

Reverse engineering
Description before unlocking: 
"What is up with these parts? Why are they shaped like that!?"

Description after unlocking:
"Ancient parts can be sacrificed for science. Scientists are twice as effective, and investigators are 4 times as effective."

History log:
"Results inconclusive. Further analysis pending."

There is another reverse engineering sprite in the old folders.

Biggest-Brian committed:
Add Investi-gate-tions sprite

Added sprite for technology "Investi-agate-tions"

Description before unlocking: 
"Temporarily commit 500 investigators to investigating the gate. Uncover its mysteries in record time!"

Description after unlocking:
"Gate investigated. We think we have a way to open it, but it's not gonna be easy... the mechanism is complicated."

History log:
"Using our reverse-engineering techniques, the investigators have come up with a theory on how to open the gate!"

Biggest-Brian committed:
.kra for 2 sprites

Added raw Krita files (.kra) for sprites for the technologies "Reverse Engineering" and "Investi-gate-tions".

Biggest-Brian commented:
Made another another two sprites for two technologies. Reverse Engineering (which I found out later that it already has an unused sprite lost among the old files) and Investi-gate-tions (which messes me up every time I type it out), both from the Abandoned Ocean. 
These are the last things I'll make today unless I discover that I screwed something up that needs important fixing.
Cheers.

Biggest-Brian commented:
Wait, does this makes it so the request can't be pulled anymore? (genuine question)

Toby222 commented:
> Wait, does this makes it so the request can't be pulled anymore? (genuine question)

Yes, you can re-open it though
If you just want to make it so it doesn't get merged until you're done, you should mark it as a draft instead

Biggest-Brian committed:
Add High-Energy Fusion sprite

Added technology High-Energy Fusion sprite
Description before unlocking: 
The lobsters are excited to show us something, but they need a lot of help first

Description after unlocking:
Improved clam-crystal fusion ration from 5-1 to 1-5. Recycler reshmycler, we have FUSION!!

History log:
The process is so bright, it could melt out our eyes out - but it's so much better than the old fusion.

Biggest-Brian committed:
Add High Energy Fusion source .kra file

Added src .kra (Krita file) for technology High Energy Fusion

Description before unlocking: 
The lobsters are excited to show us something, but they need a lot of help first

Description after unlocking:
Improved clam-crystal fusion ration from 5-1 to 1-5. Recycler reshmycler, we have FUSION!!

History log:
The process is so bright, it could melt out our eyes out - but it's so much better than the old fusion.

Biggest-Brian committed:
Fixing a file name

I forgot to take off the version number from the Reverse Engineering file name

Biggest-Brian committed:
Merge branch 'Toby222:dev' into dev

Biggest-Brian committed:
Add Cavernous Contact src .kra

Add Cavernous Contact raw .kra (Krita file)


Description before unlocking: 
The scientists have reported noises from the backside of the cave.

Description after unlocking:
Billfish can be recruited to brave the storm for more fish.

History log:
Found weird long-nose fish. Long, pointy noses... It was an awkward first encounter.

Biggest-Brian committed:
Add Cavernous Contact sprite

Add technology "Cavernous Contact" sprite. 

Description before unlocking: 
The scientists have reported noises from the backside of the cave.

Description after unlocking:
Billfish can be recruited to brave the storm for more fish.

History log:
Found weird long-nose fish. Long, pointy noses... It was an awkward first encounter.

Biggest-Brian commented:
Since my last comment, two more sprites for technologies have been added. "High-Energy Fusion" from the Marine Ocean and "Cavernous Contact" from the Tempestuous Ocean.

Toby222 commented:
:pray:

Biggest-Brian committed:
Add Sandbagging sprite

Add sandbagging .png

Description before unlocking: 
If crabs venture out too far, they risk getting lost. The billfish have a suggestion.

Description after unlocking:
We can equip crabs with sandbags to make them into stormgoers. They can venture much farther from the cave.

History log:
The billfish showed us how to tie seagrass into bundles. By filling them with sand, we can weigh down crabs!

Biggest-Brian committed:
Add Sandbagging src .kra

Added Sandbagging src .kra (Krita file)

Description before unlocking: 
If crabs venture out too far, they risk getting lost. The billfish have a suggestion.

Description after unlocking:
We can equip crabs with sandbags to make them into stormgoers. They can venture much farther from the cave.

History log:
The billfish showed us how to tie seagrass into bundles. By filling them with sand, we can weigh down crabs!

Biggest-Brian committed:
Add Assisted Extraction sprite

Add techonology "Assisted Extraction" sprite .png

Description before unlocking: 
Crabs take forever to get crystals. The squid insist that working together will help. I guess it's better than nothing.

Description after unlocking:
We may now organize crabs and squid into teams of 2 to expedite crystal extraction.

History log:
A crab can reach places a squid cannot, and a squid can help a crab get around faster. The squid were right, this is great!

Biggest-Brian committed:
Add Assisted Extraction src .kra

Addind assisted extraction .kra (Krita file)

Description before unlocking: 
Crabs take forever to get crystals. The squid insist that working together will help. I guess it's better than nothing.

Description after unlocking:
We may now organize crabs and squid into teams of 2 to expedite crystal extraction.

History log:
A crab can reach places a squid cannot, and a squid can help a crab get around faster. The squid were right, this is great!

Biggest-Brian committed:
Add sociology sprite

Added sociology .png sprite technology from Volcanic Ocean

Description before unlocking: 
What is a frenzy? Why do we obey our orders? Why do we work together?

Description after unlocking:
Shrimp and sponge farms are twice as effective now that we can grasp the idea of a social system being different than ours.

History log:
Progress, prosperity, organization, order. We choose to be a part of this society, and that is what creates it.

Biggest-Brian committed:
Delete sociology.png

I forgot to put it through oxipng

Biggest-Brian committed:
Add sociology technology spirte

Add sociology technology from the Volcanic Ocean sprite .png
Now with correct oxipng compression.

Description before unlocking: 
What is a frenzy? Why do we obey our orders? Why do we work together?

Description after unlocking:
Shrimp and sponge farms are twice as effective now that we can grasp the idea of a social system being different than ours.

History log:
Progress, prosperity, organization, order. We choose to be a part of this society, and that is what creates it.

Biggest-Brian committed:
Add sociology src .kra

Added sociology technology .kra (Krita file)

Description before unlocking: 
What is a frenzy? Why do we obey our orders? Why do we work together?

Description after unlocking:
Shrimp and sponge farms are twice as effective now that we can grasp the idea of a social system being different than ours.

History log:
Progress, prosperity, organization, order. We choose to be a part of this society, and that is what creates it.

Biggest-Brian committed:
Add crystal spear sprite

Added crystal spear sprite.

Description before unlocking: 
Our spear-wielding friends could use an upgrade.

Description after unlocking:
Billfish catch fish two and a half times faster thanks to a diversified toolset.

History log:
We've designed various pointy implements for billfish to use. A sharper slash is a better slash.

Biggest-Brian committed:
Add crystal spear src .kra

Added crystal spear technology from the Tempestuous Ocean .kra (Krita file)

Description before unlocking: 
Our spear-wielding friends could use an upgrade.

Description after unlocking:
Billfish catch fish two and a half times faster thanks to a diversified toolset.

History log:
We've designed various pointy implements for billfish to use. A sharper slash is a better slash.

Biggest-Brian commented:
Since last comment, I've added more 4 sprites for technologies. 
Sandbagging: Tempestuous Ocean
Assisted Extration: Frigid Ocean
Sociology: Volcanic Ocean
Crystal Spear: Tempestuous Ocean

God bless soluble cappuccino

Biggest-Brian committed:
Add Crevice Contemplation sprite

Added technology "Crevice Contemplation" sprite .png

Crevice Contemplation

Description before buying:
What's in the holes that the eels dig? Why do they make them? How DOES an ell pit work?

Description after buying:
Eels are twice as effective, and so are eel pits. We learned some things we'd rather not have, but I guess they were just trying to help. I guess.

Log:
All of our questions and more were answered by the antsy eels. We didn't need to know that much. Thanks, though, I guess??

Biggest-Brian committed:
Add Crevice contemplation src .kra

Added Crevice Contemplation technology .kra (Krita file)

Crevice Contemplation

Description before buying:
What's in the holes that the eels dig? Why do they make them? How DOES an ell pit work?

Description after buying:
Eels are twice as effective, and so are eel pits. We learned some things we'd rather not have, but I guess they were just trying to help. I guess.

Log:
All of our questions and more were answered by the antsy eels. We didn't need to know that much. Thanks, though, I guess??

Biggest-Brian committed:
Mess with the sprites.js

I have tried to add the sprite from the technology "Crevice Contemplation"
Pray to your lords.

Biggest-Brian committed:
Add sprites to sprites.png

See last change

Biggest-Brian committed:
I put some sponge, jellyfish, ice and calcinium facts

Biggest-Brian committed:
Added two fun facts about collector crabs

Biggest-Brian committed:
Implement added sprites to the game

Added all the other sprites I've previously added as .pngs in the sprites.png to the .js

I hope it doesn't break anything

Biggest-Brian commented:
Okay, it's added in the .js
I don't know if what I did was right, and don't know how to check it.

(I also added more fun facts)

Biggest-Brian committed:
Added curious collection sprite

Added curious collection sprite

Description before buying:
We've been collecting all this 'coral' stuff. What actually IS it?

Description after buying:
We now have the most absolutely rudimentary understanding of coral. Crabs and curious crabs work twice as efficiently because of this.

Log:
It's an animal of some kind, since it moves. It clearly eats something, but we have no idea what. It just kinda reaches into the water sometimes. Weird.

Biggest-Brian committed:
Add four sprites .kra

Added .kra (Krita file) sprites for the technologies Enviromentalism, Calcinium Studies, Curious Collection and Calcinium Robotics.

Biggest-Brian committed:
Add Calcinium Robotics technology sprite

Description before buying:
So the lobsters remembered what calcinium is for.

Description after buying:
Rays and planter crabs can be trained as clam scavengers and seabed strippers. These tools are designed to work without input from their users. Freaky.

Log:
With enough effort, we've managed to fashion calcinium tools for the frenzy.

Biggest-Brian committed:
Change Calcinium Robotics sprite

I didn't like the shading.

Calcinium Robotics

Description before buying:
So the lobsters remembered what calcinium is for.

Description after buying:
Rays and planter crabs can be trained as clam scavengers and seabed strippers. These tools are designed to work without input from their users. Freaky.

Log:
With enough effort, we've managed to fashion calcinium tools for the frenzy.

Biggest-Brian committed:
Add Enviromentalism sprite

Added Enviromentalism technology sprite

Description before buying:
The machines produce what now?! Quick, we need a solution - sponges filter stuff, right?!

Description after buying:
Sponges can be turned into filters to stop the tar from killing us all. Yay!

Log:
With the right kind of stretching and squishing, we can turn sponges into weird little filter things!

Biggest-Brian committed:
Add Calcinium Studies sprite

Added Calcinium Studies technology sprite

Description before buying:
The lobsters have seen out transmutation process, and they say that it reminds of something?

Description after buying:
Learned how to manufacture calcinium. I wonder what it's for? The lobsters are currently consolidating their information on it.

Log:
And there we have it. Calcinium, I guess! Surprisingly strong.

Biggest-Brian committed:
Add 22 new sprites to the sheet

The following sprites have been added to sprites.png with the site https://free-tex-packer.com/app/. 



-Actions (courtesy of Glowkate):

Get Squid collective
Get Dolphin historian



-Aspects:

.(made by BiggestBrian):
Destiny Gamble
Overtime
.(made by Glowkate):
Distant Foresight
Meditation
The Plan



-Technologies

.Abandoned:
Enviromentalism
Investi-gate-tions
Octal efficiency
Octopus Methodology

.Frigid:
Assisted Extraction

.Haven (courtesy of Glowkate):
Retroactive Recordkeeping

.Marine:
Calcinium Studies
Calcinium Robotics
High-energy Fusion

.Shrouded:
Crevice Contemplation

.Tempestuous:
Cavernous contact
Crystal Spear
Sandbagging

.Volcanic:
Curious Collection
Sociology

Biggest-Brian committed:
Put the added sprites from the sheet in the .js

The following sprites have been added to sprites.js with the site https://free-tex-packer.com/app/. 



-Actions (courtesy of Glowkate):

Get Squid collective
Get Dolphin historian



-Aspects:

.(made by BiggestBrian):
Destiny Gamble
Overtime
.(made by Glowkate):
Distant Foresight
Meditation
The Plan



-Technologies

.Abandoned:
Enviromentalism
Investi-gate-tions
Octal efficiency
Octopus Methodology

.Frigid:
Assisted Extraction

.Haven (courtesy of Glowkate):
Retroactive Recordkeeping

.Marine:
Calcinium Studies
Calcinium Robotics
High-energy Fusion
Shrouded:
Crevice Contemplation

.Tempestuous:
Cavernous contact
Crystal Spear
Sandbagging

.Volcanic:
Curious Collection
Sociology

Toby222 commented on js/facts.js:
```diff
@@ -189,7 +194,12 @@ SharkGame.FunFacts = {
             // "Sharks would definitely not have a way of acquiring most kinds of jellyfish in real life.",
             "Jellyfish can be extremely dangerous. Some kinds of box jellyfish have fatal stings.",
             "Turritopsis dohrnii is a species of jellyfish that can restart its lifecycle at will. In theory, this grants it an infinite lifespan.",
+            "Jellyfishes are very old, evolutionarily speaking. A few jellyfish fossils have been dated to approximately 500 million years ago.",
+            "The gastric system of jellyfishes has only one hole, which means that food comes out from the same place as it goes in. Ewwww.",
+            "Jellyfishes are from the 'cnidaria' phylum, the same phylum of sea anemones.",
```
Use "jellyfish" as plural for consistency with the others

Toby222 commented on js/facts.js:
```diff
@@ -219,13 +230,16 @@ SharkGame.FunFacts = {
             "Sponge is not a plant, it is an animal. A weird, amorphous animal.",
             "The pores in sponges are designed to help them filter water for food at maximum efficiency.",
             "Many species of sponge have a mualistic realitionship with species of algae. The algaes use photosynthesis to produce food for the sponges.",
+            "Sponges have bacteria inside their own cells that help with the metabolism of many substances. This is called 'endosynbiosis'.",
```
endosy`m`biosis

Biggest-Brian committed:
Fix the sprites.js

The last change I done made every sprite stop appearing. So I'm trying again.

Biggest-Brian committed:
Fix sprites.js again

I broke it again, third time is the charm

Biggest-Brian committed:
Fix sprites.js again again

This time it will work, trust

Biggest-Brian committed:
Fix sprites.js a fourth time

If this doesn't work, I officially don't know what to do.
I don't even know how to rollback a commit.
Thank god I'm not a programmer.

Biggest-Brian committed:
Put the sprites made in the sheet

The following sprites have been added to sprites.png with the site https://free-tex-packer.com/app/. 



-Actions (courtesy of Glowkate):

Get Squid collective
Get Dolphin historian



-Aspects:

.(made by BiggestBrian):
Destiny Gamble
Overtime
.(made by Glowkate):
Distant Foresight
Meditation
The Plan



-Technologies

.Abandoned:
Enviromentalism
Investi-gate-tions
Octal efficiency
Octopus Methodology

.Frigid:
Assisted Extraction

.Haven (courtesy of Glowkate):
Retroactive Recordkeeping

.Marine:
Calcinium Studies
Calcinium Robotics
High-energy Fusion

.Shrouded:
Crevice Contemplation

.Tempestuous:
Cavernous contact
Crystal Spear
Sandbagging

.Volcanic:
Curious Collection
Sociology

Biggest-Brian committed:
Fix a few things

Fix typos, lack of ] and more typos.

Biggest-Brian commented:
Added 6 sprites made by Glowkate in the sheet, those being:
Retroactive Recordkeeping (Haven Ocean technology) 
The plan (Aspect)
Meditation (Aspect)
Distant Foresight (Aspect)
Get Historian (Haven Ocean action)
Get Collective (Abandoned Ocean action)

So this is a reminder to credit them as well

Biggest-Brian committed:
Add billfishBiology .kra src

Added Billfish Biology .kra (Krita file)

Description before buying:
We still don't know much about our lance-wielding friends. They zip around in the water and slash at stuff, but what are they REALLY?

Description after buying:
Billfish are four times more effective, and billfish pairs are available. I swear, our scientists must be losing it - these things are not fish, surely.

Log:
We refuse to accept the narrative the billfish are pushing on us, but the scientists are adamant. They say they are literally just fish with big noses.

Biggest-Brian committed:
Add Billfish Biology sprite .png

Billfish Biology from Tempestuous Ocean

Description before buying:
We still don't know much about our lance-wielding friends. They zip around in the water and slash at stuff, but what are they REALLY?

Description after buying:
Billfish are four times more effective, and billfish pairs are available. I swear, our scientists must be losing it - these things are not fish, surely.

Log:
We refuse to accept the narrative the billfish are pushing on us, but the scientists are adamant. They say they are literally just fish with big noses.

Biggest-Brian committed:
Making a To do list

Gosh, there is quite a bit to do, huh?

Biggest-Brian committed:
Add Laser Lenses .kra

Add Laser Lenses .kra (Krita file) in src

Biggest-Brian committed:
Add laser lenses sprite

Add tempestuous ocean technology laser lenses sprite

Biggest-Brian committed:
Add Infinity Vision .kra

Add aspect Infinity Vision .kra

Reveal all aspects which are not locked
"Nothing can stay hidden to astute observers"

Biggest-Brian committed:
Add Infinity Vision sprite

Add aspect "Infinity Vision" .png

Reveal all aspects which are not locked
"Nothing can stay hidden to astute observers"

Biggest-Brian committed:
Otimize size using oxipng

Otimize sprite sheet size using oxipng, 213KB (37,08% smaller)

Biggest-Brian committed:
Add sprites for several aspects

Added the .png sprites for infinity vision, mechanical manifestation and collective cooperation. All have been smalled by oxipng

Biggest-Brian committed:
Otimize pngs with oxipng

Otimize Billfish Biology and Laser Lenses technologies pngs with oxipng

Biggest-Brian committed:
Updated facts

Put a sharkonium fact (replaced one which sentence was incompleted) and put some calcinium converter facts (more like machine-brain interface facts, actually)

Biggest-Brian committed:
Add Calcinium Cybernetics sprite

Added calcinuium cybernetics sprite .png

Description before buying:
Robot tools are great and all, but they're pretty dumb on their own. We could od better if we learned to control them.

Description after buying:
Clam scavengers are twice as effective and seabed strippers are four times as effective. Also, we can make calcinium converters??

Log:
A bundle of crumpled blueprints and a few morally dubious experiments later, brains can now interface with calcinium machines. Future!

Biggest-Brian committed:
Add Calcinium Cybernetics src .kra

Added Calcinium Cybernetics marine technology .kra (Krita file)

Biggest-Brian committed:
Update To Do.md

Biggest-Brian committed:
Update To Do.md

Biggest-Brian committed:
Updating sprites.png

Adding the following sprites to sprites.png
- Volcanic Ocean technology "Curious Collection" (by Glowkate)
- Marine Ocean technology "Bioengineering"
- Tempestuous Ocean technology "Billfish Biology"
- Tempestuous Ocean technology "Laser Lenses"
- Aspect "Infinity Vision"
- Aspect "mechanical manifestation"
- Aspect "collective cooperation"
- Marine Ocean technology "Calcinium Cybernetics"

Biggest-Brian committed:
Update sprites.js

I sure hope I don't need to fix this thing four times in a row this time.

Adding the following sprites to sprites.js
- Volcanic Ocean technology "Curious Collection" (by Glowkate)
- Marine Ocean technology "Bioengineering"
- Tempestuous Ocean technology "Billfish Biology"
- Tempestuous Ocean technology "Laser Lenses"
- Aspect "Infinity Vision"
- Aspect "mechanical manifestation"
- Aspect "collective cooperation"
- Marine Ocean technology "Calcinium Cybernetics"

Biggest-Brian committed:
Fix sprites.js

You know, it's funny. You have to take off all the .png from the code.

Biggest-Brian committed:
"Fixed" Far Exploration's

Haven Ocean and Abandoned Ocean had their technologies "Far Exploration" named differently for no apparent reason, so I equalized all of them.

Biggest-Brian reviewed (commented):
I screwed up

Toby222 commented:
> I screwed up

how do you mean?

Toby222 commented:
oh I see

Biggest-Brian committed:
Actually fixing Far Exploration

My last fix broke stuff

Biggest-Brian commented:
Okay, crisis over, it's fixed
Broken: 
![image](https://github.com/user-attachments/assets/a9786d5e-ceff-4192-894c-9aa31eddbf01)
Fixed:
![image](https://github.com/user-attachments/assets/15299018-bbaa-4b4f-af44-85f93a1b54cc)

Biggest-Brian committed:
Update To Do.md

Biggest-Brian committed:
sprites.js

this will be fixed later

Biggest-Brian committed:
sprites.png

this doesn't need to be fixed

Biggest-Brian committed:
Update sprites.js

Biggest-Brian committed:
Fixing names

NOTE: Technologies which names have hifens do not have hifens in its secret name

Biggest-Brian commented:
Oh hey, look! I forgot and left a lot of Glowkate's sprites out of my last sprites.js and sprites.png update. Too bad, I won't be fixing this today! Good night.

Biggest-Brian committed:
Update sprites.png with Glowkate's sprites

Biggest-Brian committed:
Update sprites.js with Glowkate's sprite

Biggest-Brian committed:
Update To Do.md

Biggest-Brian committed:
Fix sprites.js

Taking off the .png from the sprite names

Biggest-Brian committed:
Add two sprite .kra

Adding arcane sifting and ancestral recall .kra (Krita file)

Biggest-Brian committed:
Adding two sprites

Added sprites .png for two Shrouded Ocean technologies

Arcane Sifting
Description before buying:
An eel just came back with a tiny piece of arcana-looking rubble, excitedly yelling about how they found it in the sand. Are we onto something here?

Description after buying:
Can train eels to sift through sand in search of arcana.

Log:
Eels tasked with sifting through the sand have come back with even more arcana. We should keep this up!

Ancestral recall
Description before buying:
The sharks and rays, and even chimaeras, know we share some features among ourselves. Using the strange tales passed down by chimaeras, let's piece together the puzzle.

Description after buying:
Sharks, rays and chimaeras, and their roles, are all four times as effective. Except divers. They're 16 times as effective. We have had a glorious past. Now, on to a glorious future.

Log:
These tales speak plainly about a frenzy not unlike our own. It's said that what came of it was glorious, but all of the specifics have been lost to time, it seems.

Biggest-Brian committed:
Update To Do.md

Biggest-Brian committed:
Update To Do.md

I finished all the aspects that were not yet made

Biggest-Brian committed:
Optimize and add sprites

I maybe forgotten to optimize some sprites I've added before, so I'm optimized all aspect sprites and uploaded all of them again.

Adding the aspect sprites I've made today (after snorting a line of soluble cappuccino (don't do this at home))

Added aspect sprites: In-game 
The Dial: Unlock the choice to slow down time in exchange for a large multiplier on Patience rewards.
Coordinated Cooperation: Tokens increase production by x
Clean Slate: Unlock the ability to refund some aspects.
Gumption: For each unspent essence you have, increase non-patience essence gains by %. Effect caps at 100%.
Token of Industry: Unlock token (tokens cannot stack on the same resource).

Biggest-Brian committed:
Adding five .kra src

Adding the .kra (Krita files) for the following aspects

The dial
Token of industry
Clean Slate
Gumption
Coordinated cooperation

Biggest-Brian committed:
Update sprites.png

Put some more sprites, I don't even remember which

Biggest-Brian committed:
Update sprites.js

Updating sprites.js

Biggest-Brian committed:
Fix sprites.js

Biggest-Brian committed:
Compressing sprites.png with oxipng

I forgot to compress sprites.png when I uploaded it last change. So I'm compressing it now.

Biggest-Brian committed:
Add Pack Hunting sprite (looking at fimsh)

Added Pack Hunting Volcanic Ocean technolgy .png

Description before buying:
Convince rays to hunt in packs for higher catch efficiency.

Description after buying:
Hunting in groups makes rays twice as effective. Shark strategy is efficient.

Log:
By hunting in groups, rays can now effectively target large schools of fish before they dart away.

Biggest-Brian committed:
Add Pack Hunting src .kra

Added Volcanic Ocean technology Pack Hunting .kra (krita file)

Biggest-Brian committed:
Update To Do.md

Biggest-Brian committed:
Add imperial designs sprite

Added Imperial Designs .png sprites

Imperial Designs

Before unlocking: Finally, we've found them! After rummaging through the kelp papyrus for a bit, we came across the original designs for the dolphin machines.

After unlocking: Kelp cultivators and crimson combines are 4 times faster, and tireless crafters are four times as efficient. We begrudingly admit their quality is not entirely terrible.

Log: Upon further examination, these suck! These designs will never work! Look, let's show them-- oh. Oh, apparently they do. Huh.

Biggest-Brian committed:
Add imperial designs src .kra

Added imperial designs .kra file (Krita file)

Biggest-Brian committed:
Add consistent Communication sprite

Added volcanic technology Consistent Coomunication .png

Description before buying:
Okay shrimps, look! We have algae to make food for sponges! We don't need your sponges anymore!

Description after buying:
Can now hire shrimp, though they're kinda mean. How were we supposed to know that sponge needs algae for some reason?

Log:
With ample demonstration of what we've gathered through farming, a few shrimp are convinced of our good intentions.

Biggest-Brian committed:
Add consistent communication src .kra

Added Volcanic technology Consistent Communication .kra (Krita file)

Biggest-Brian committed:
Fixing to do

Updated To Do wbecause I've made Crystalline Construction, and put Chasm Exploration, that is without a sprite and wasn't in the list.

Biggest-Brian committed:
Add Crystalline Construction sprite

Added Crystalline Construction Haven Ocean techonology .png

Crystalline Construction

Before unlocking: The dolphins are a bunch of jerks, but maybe we can still learn from one another. Maybe.

After unlocking: All dolphin machines run eight times as fast, except tireless crafters, which are 8 times as efficient. Holy moley! Also, shark science is way more informative now that we have more perspective, so scientists are 16 times faster.

Log: By integrating shark science with dolphin design, we've managed to create a superior set of machines. Maybe we work better together than we do apart.

Biggest-Brian committed:
Add Glowkate's sprites

I don't actually know if Glowkate put this in their own merge request, so I'm putting here just in case.
Added the sprites for:
Shrouded Ocean's Arcane Sacrifice tech
Marine Ocean's Mobius Shells tech
Volcanic Ocean's Curious Collection tech

Biggest-Brian committed:
Add dolphin Technoloy sprite

Biggest-Brian committed:
Ancient Agriculture sprite

Biggest-Brian committed:
Add src .kra to the things I've made and added

Biggest-Brian committed:
Update sprites.js

I'll fix this

Biggest-Brian committed:
Update sprites.js

I have fixed this

Biggest-Brian committed:
Updating sprites.png

Hopefully everything is right

Biggest-Brian committed:
Just testing things, will change them back if it causes problems

Changed farHavenExploration and farAbandonedExploration to just "farExploration" in hope of fixing something I broke.

Biggest-Brian commented:
It seems to have worked
This should not have worked.

Biggest-Brian committed:
Just fixing small stuff 

There were pixels that shouldn't been there.

Biggest-Brian committed:
Sprites.png

I had to fix some sprites

Biggest-Brian committed:
I am typing letters

Biggest-Brian committed:
Fixing sprites.js

Biggest-Brian committed:
Update sprites.js

Biggest-Brian commented:
Cheers

Biggest-Brian committed:
Fixing To Do.md

Eightfold Optimization was not made

Biggest-Brian commented:
Oh yeah. I just remembered. I didn't add all the sprites that Glowkate made in the folders in img/small, despite them being in the sprites.js and sprites.png because they're already in Glowkate's pull request. Should I add them here?
(Also, I forgot to count two sprites of theirs, don't mind the title change)
